slurp/src/main/webapp/index.html

```diff
@@ -17,6 +19,19 @@
     <script defer src="scripts/init-firebase.js"></script>
     <script defer src="scripts/script.js"></script>
   </head>
+
   <body>
+
+    <div id="login-card">
+      <div class="login-title">
+        <h1 id="welcome">Welcome to</h1>
+        <h1 id="slurp">SLURP</h1>
+      </div>
+      <!-- TODO: Put path to logo when we have one. -->
```

Comment:
May want to create an issue for this. I am not sure if this is too small for creating an issue but I think that is best practice to create an issue for any todo.

Reply:
+1

Reply:
+1
Please open an issue for this and reference the issue in the TODO.

Reply:
Done, good thing to keep in mind for the future too.